src/Core/src/Handlers/Editor/EditorHandler.Android.cs

```diff
@@ -95,6 +95,23 @@ void OnPlatformViewAttachedToWindow(object? sender, ViewAttachedToWindowEventArg
 		}
 
 		void OnTextChanged(object? sender, Android.Text.TextChangedEventArgs e) =>
-			VirtualView?.UpdateText(e);
+			VirtualView?.UpdateText(e); 
+		
+		void OnFocusedChange(object? sender, FocusChangeEventArgs e)
+		{
+			var platformView = sender as AppCompatEditText;
+
+			if (e.HasFocus)
+			{
+				// Post this to the main looper queue so it doesn't happen until the other focus stuff has resolved
+				// Otherwise, ShowKeyboard will be called before this control is truly focused, and we will potentially
+				// be displaying the wrong keyboard
+				platformView?.PostShowKeyboard();
+			}
+			else
+			{
+				platformView?.HideKeyboard();
```

Comment:
What happens here if you are tabbing through multiple fields?

Reply:
If we have something like:

StackLayout
- Entry
- Entry
- Entry

If we have the focus on the first entry, and pass the focus to the second one:
- The first Entry loses focus and the keyboard closes.
- The second Entry gains focus so the keyboard is reopened.

That is, the keyboard would close and open in a short time.

Reply:
I don't feel like that's the behavior we'd want here. If we want to maintain some level of behavior here with MAUI then I would say at the controls level if the user calls "FOCUS" then we open the keyboard but then that's it.